### ISSUE TITLE
correct torrent traffic?

### ISSUE THREAD

**Issue by rezad1393** [jow-/nlbwmon#18]
how can I configure the setting to get correct torrent traffic?
I have added the port set in deluge torrent client in nlbwmon but an small traffic is logged as deluge-tcp  and most is counted as others.
the port mapping I've put is 
"6 1803 deluge-tcp"
"17 1803 deluge-udp"
should I have put something else?

is there anyway to see the traffic that is counted as other in port ip protocol format? so that I can classify them easily?

**Comment by jow-** [comment]
You cannot classify torrent traffic by the means of port numbers.

**Comment by rezad1393** [comment]
why? I have put the port number for it in torrent client
are you telling me that torrent client connects on another port?

**Comment by jow-** [comment]
The port number is usually just used for control traffic. The actual peer to peer data transfers use random UDP ports for each connection.

**Comment by rezad1393** [comment]
cant I set that to be not random?
in deluge I can set both incoming and outgoing port.
EDIT: I have put the port for outgoing same as incoming (1803)and now it seems to work partially.
the deluge-udp works but I still get a lot of other protocol that is almost half of deluge-udp.

maybe it is imortant to tell you that I run lede on my router/switch and it is bridged with a modem conning via adsl.
maybe that is messing with the reading.

UPDATE:
it is deluge fault
I tested with rtorrent and that client uses the ports assigned (at least with the torrent I tested)

second update.
nope. it seems even rtorrent traffic doesnt get counted
